server/docker-entrypoint.sh

```diff
@@ -195,9 +195,8 @@ RUN_TYPE='STANDALONE'
 CONTAINER_SETTINGS="true"
 SERVER_OPTIONS=""
 SERVER_CONFIGURATION="clustered.xml"
-# Xms should always be set: https://developers.redhat.com/blog/2014/07/15/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-1/
-JAVA_OPTS="-Xms64m -Djava.net.preferIPv4Stack=true"
-PERCENT_OF_MEMORY_FOR_MX=70
+JAVA_OPTS="-Djava.net.preferIPv4Stack=true -XX:+DisableExplicitGC"
+PERCENT_OF_MEMORY_FOR_MX=50
```

Comment:
Exactly... You can always specify Xmx on your own and the autoscaling script will back off. See https://github.com/jboss-dockerfiles/infinispan/blob/master/server/docker-entrypoint.sh#L309. 

But you know what.. let me write it explicitly into the manual.